Makefile

```diff
@@ -6,6 +6,18 @@ else
 	SECCOMP =
 endif
 
+#
+# We differentiate between release / debug build types using the BUILD_TYPE
+# environment variable.
+#
+ifeq ($(BUILD_TYPE),release)
```

Comment:
This env variable should probably also affect how we build the rust parts when you build everything with just `make`. We don't do that in the CI, we build the postgres and rust parts separately, so this PR is enough for the CI.

(Anyone remember why we build postgres and rust parts separately in the CI? Why not just call `make`? Is it because of how the build caching works?)

Reply:
> Is it because of how the build caching works?

Yes, I think that was the reason. That way also rust changes would not trigger postgres rebuild.